src/ui/find-ui/tests/FamilyHubs.ServiceDirectory.UnitTests/Filtering/SearchWithinFilterTests.cs

```diff
@@ -0,0 +1,27 @@
+using FamilyHubs.ServiceDirectory.Core.ServiceDirectory.Models;
+using FamilyHubs.ServiceDirectory.Web.Filtering.Filters;
+
+namespace FamilyHubs.ServiceDirectory.UnitTests.Filtering;
+
+public class SearchWithinFilterTests
+{
+    private readonly ServicesParams _mServicesParams = new("", 0, 0);
+
+    [Fact]
+    private void TestFilterNone()
+    {
+        var filter = new SearchWithinFilter();
+        filter.AddFilterCriteria([], _mServicesParams);
+
+        Assert.Null(_mServicesParams.MaximumProximityMeters);
+    }
+
+    [Fact]
+    private void TestFilter()
```

Comment:
This may just be opinionated: The test names are not descriptive in what it's testing. Behaviorial language is a good place to go, like 'ShouldReturnNullIfFilterIsEmpty'
I would apply this comment to all the tests starting with 'Test'